packages/shared/lib/services/notification/slack.service.ts

```diff
@@ -303,7 +310,7 @@ class SlackService {
             return;
         }
 
-        const actionResponse = await syncClient.triggerAction<SlackActionResponse>({
+        const actionResponse = await this.orchestrator.triggerAction<SlackActionResponse>({
```

Comment:
I guess using an action to post to Slack is dog fooding but is there any other reason why sending Slack notification is done like this.

Reply:
I feel like it's handy for oauth/token reason

Reply:
would a `proxy` call achieve the same result without having to run a script?